src/api/routes/group.ts

```diff
@@ -0,0 +1,74 @@
+import { Router } from "express";
+import { Container } from "typedi";
+import GroupService from "../../services/GroupService";
+
+const groupRouter = Router();
+const groupService = Container.get(GroupService);
+
+const handler = async (cb: () => Promise<void>, res) => {
+    try {
+        await cb();
+    } catch(error) {
+        res.status(400).json({ error: { message: error.message } });
```

Comment:
400 is a error status code of user mistake. It's better to use 50x errors if you don't know the reason of the error

Reply:
Done.